integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala

```diff
@@ -101,6 +100,12 @@ class CarbonSpark2SqlParser extends CarbonDDLSqlParser {
     loadDataNew | explainPlan | alterTableColumnRenameAndModifyDataType |
     alterTableAddColumns
 
+  protected lazy val materializedViewCommands: Parser[LogicalPlan] =
```

Comment:
As already a new Parser => `MVParser` is already present. Why it is added?

Reply:
MVParser will be delete when clean the old mv implementation